launchable/commands/inspect/subset.py

```diff
@@ -54,7 +54,7 @@ def convert_row(list: List[Dict], order: int, is_subset: bool):
     order: start number of order
     is_subset: in subset or not
     """
-    return [[order + i, "#".join([path["type"] + "=" + path["name"]
-                                  for path in l["testPath"]]), "✔"
-             if is_subset else "", "{:0.4f}".format(l["duration"] / 1000)]
+    return [[order + i, "#".join([path.get("type", "") + "=" + path.get("name", "")
```

Comment:
Seems this assumes testPath potentially does not contain type and name. Is it acceptable to create a string like `#=` if the testPaths don't have the keys?

Reply:
thanks.
fixed to skip if the key does not exist
https://github.com/launchableinc/cli/pull/378/commits/733db94932515a75654e6fb4c8ef3ceb3244e6cb